phylogenetic/config/description.md

```diff
@@ -0,0 +1,14 @@
+We gratefully acknowledge the authors, originating and submitting laboratories of the genetic sequences and metadata for sharing their work. Please note that although data generators have generously shared data in an open fashion, that does not mean there should be free license to publish on this data. Data generators should be cited where possible and collaborations should be sought in some circumstances. Please try to avoid scooping someone else's work. Reach out if uncertain.
+
+We curate sequence data and metadata from NCBI as starting point for our analyses. Curated sequences and metadata are available as flat files at:
+
+* data.nextstrain.org/files/workflows/dengue/sequences_all.fasta.zst
+* data.nextstrain.org/files/workflows/dengue/metadata_all.tsv.zst
+* data.nextstrain.org/files/workflows/dengue/sequences_denv1.fasta.zst
+* data.nextstrain.org/files/workflows/dengue/metadata_denv1.tsv.zst
+* data.nextstrain.org/files/workflows/dengue/sequences_denv2.fasta.zst
+* data.nextstrain.org/files/workflows/dengue/metadata_denv2.tsv.zst
+* data.nextstrain.org/files/workflows/dengue/sequences_denv3.fasta.zst
+* data.nextstrain.org/files/workflows/dengue/metadata_denv3.tsv.zst
+* data.nextstrain.org/files/workflows/dengue/sequences_denv4.fasta.zst
+* data.nextstrain.org/files/workflows/dengue/metadata_denv4.tsv.zst
```

Comment:
It would be helpful to format these as Markdown links so that they can be used to directly download the data. 

E.g.
```
- [data.nextstrain.org/files/workflows/dengue/sequences_all.fasta.zst](https://data.nextstrain.org/files/workflows/dengue/sequences_all.fasta.zst)
```

Reply:
Good to know! Fixed in https://github.com/nextstrain/dengue/pull/71/commits/239238896dfd12beae930317ef459669865864de